### PR TITLE
fix: update ask-sdk-core versions and fix vulnerabilities

### PR DESCRIPTION
**Pull request by Shreyas-vgr**
## Description
- Update ask-sdk-core versions and fix vulnerabilities
- Fix Github Action workflows

## Testing
- npm run test success

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs(Add new document content)
- [ ] Translate Docs(Translate document content)

## Checklist

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have read the **README** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License
- [x] By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

[issues]: https://https://github.com/alexa/ask-sdk-controls/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
         "eslint-plugin-tsdoc": "^0.2.5",
         "highlight.js": ">=10.4.1",
         "husky": "^4.2.5",
+        "marked": "^4.0.12",
```
Comment (tkm22) [inline package.json:63]: what is this for?

**Review by tkm22 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (doiron) [reply package.json:63]: seems related to markdown https://www.npmjs.com/package/marked

**Review by doiron — APPROVED** [review]
LGTM, also curious why we are adding that new dependency as Kaiming commented.

File: package.json (hunk 1 above)
Comment (Shreyas-vgr) [reply package.json:63]: https://github.com/alexa/ask-sdk-controls/security/dependabot/4

Yes it was to remove this vulnerability

**Pull request merged**